db/seed.js

```diff
@@ -42,23 +40,71 @@ const seedBatch = (minId, maxId, collection) => (
   })
 );
 
-const seedDb = async (num, collection) => {
+const printRunTime = (startTime, startId, endId) => {
+  const runTime = (new Date().getTime() - startTime) / 1000;
+  console.log(`Worker ${process.pid} done in ${runTime} sec: ids ${startId}-${endId - 1}`);
+};
+
+const seedDb = async (startId, endId) => {
+  const startTime = new Date().getTime();
+
   try {
-    for (let i = 0; i < num; i += batchSize) {
-      await seedBatch(i, i + batchSize, collection);
+    const { client, collection } = await connectToDb();
```

Comment:
Really like the way you modularized your database connection functionality here!